openslides/motions/static/js/motions/site.js

```diff
@@ -1139,6 +1139,51 @@ angular.module('OpenSlidesApp.motions.site', [
         }
         $scope.amendments = Motion.filter({parent_id: motion.id});
 
+        /** Motion comments inline editing **/
+        $scope.commentsInlineEditing = [];
+        _.forEach($scope.commentsFields, function (field) {
+            var inlineEditing = MotionInlineEditing.createInstance($scope, motion,
+                'view-original-comment-inline-editor-' + field.name, false,
+                function (obj) {
+                    return motion['comment ' + field.name];
+                },
+                function (obj) {
+                    motion['comment ' + field.name] = obj.editor.getData();
+                }
+            );
+            $scope.commentsInlineEditing.push(inlineEditing);
+        });
+        $scope.commentsInlineEditingSaveToolbarVisible = function () {
```

Comment:
Can we please find a solution not to put these stuff into the "global" scope of this controller? The `$scope.commentsInlineEditing` can be a container that hold all general control functions (like this one) and all inlineEditing instances which contains themselves control functions for single comment fields (like save, revert, enable, disable, ... I think).